packages/components/src/components/data-table/_data-table-expandable.scss

```diff
@@ -64,36 +64,12 @@
 
   tr.#{$prefix}--parent-row.#{$prefix}--expandable-row + tr[data-child-row] td {
     padding-left: 4rem;
-    border-bottom: 1px solid $ui-03;
+    border-bottom: 1px solid $layer-alt;
     transition: padding-bottom $duration--fast-02 motion(standard, productive),
       transform $duration--fast-02 motion(standard, productive),
       background-color $duration--fast-02 motion(standard, productive);
   }
 
-  //compact child row padding
```

Comment:
@jnm2377 I noticed these were added in https://github.com/carbon-design-system/carbon/pull/8080, but it seemed to cause misalignment issues in the size variants of the data table. When I remove these styles, they all align with the parent column, but I was unsure if this was an intended visual update

<img width="371" alt="Screen Shot 2021-04-01 at 10 38 46 AM" src="https://user-images.githubusercontent.com/11928039/113310659-7942a700-92d6-11eb-9b14-9817307d71a8.png">
<img width="374" alt="Screen Shot 2021-04-01 at 10 38 40 AM" src="https://user-images.githubusercontent.com/11928039/113310663-79db3d80-92d6-11eb-8ad3-d8b985801133.png">
<img width="362" alt="Screen Shot 2021-04-01 at 10 38 34 AM" src="https://user-images.githubusercontent.com/11928039/113310664-79db3d80-92d6-11eb-9f5b-17f2645d16e6.png">
<img width="550" alt="Screen Shot 2021-04-01 at 10 38 28 AM" src="https://user-images.githubusercontent.com/11928039/113310666-7a73d400-92d6-11eb-8222-12a2ee76f142.png">

Reply:
The changes were intentional, but I think I added these styles before I addressed this comment: https://github.com/carbon-design-system/carbon/pull/8080#pullrequestreview-612344005 and then forgot to recalculate the padding after the fix. Lemme mess around a bit and see if we still need them. Or figure out what the correct padding should be.

Reply:
@tw15egan you're right! These aren't needed after I addressed that comment lol AH! Thanks for that catch!!! Do we want to just add the same padding for all the sizes here? Rather than in the storybook? Or is the padding optional and we just add it in the storybook for prettiness? cc: @aagonzales

Reply:
This PR will normalize the left-padding between all size variants